### DISCUSSION TITLE
Welcome to gTTS Discussions!

### DISCUSSION THREAD

**Discussion by pndurette** [pndurette/gTTS#272] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by FennecCoder** [comment]
Is there anyway to change voice from female to male , please!
